main.go

```diff
@@ -16,29 +16,39 @@
 package main
 
 import (
+	"encoding/json"
 	"fmt"
 	"log"
 	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/ossf/scorecard-webapp/signing"
 )
 
 func main() {
-	http.HandleFunc("/", httpHandler)
 	fmt.Printf("Starting HTTP server on port 8080 ...\n")
+
+	r := mux.NewRouter().StrictSlash(true)
+	r.HandleFunc("/", homepage)
+	r.HandleFunc("/projects/{host}/{orgName}/{repoName}", signing.GetResults).Methods("GET")
+	http.Handle("/", r)
+
 	if err := http.ListenAndServe(":8080", nil); err != nil {
 		log.Fatal(err)
 	}
 }
 
-func httpHandler(w http.ResponseWriter, r *http.Request) {
-	switch r.Method {
-	case http.MethodGet:
-		w.WriteHeader(http.StatusOK)
-		if _, err := w.Write([]byte("Hello world!" +
-			" This site is still under construction." +
-			" Please check back again later.")); err != nil {
-			log.Printf("error during Write: %v", err)
-		}
-	default:
-		http.Error(w, "only GET method is allowed", http.StatusMethodNotAllowed)
+func homepage(w http.ResponseWriter, r *http.Request) {
+	endpts := struct {
+		GetRepoResults string `json:"get_repo_results"`
+	}{
+		GetRepoResults: "https://api.securityscorecards.dev/projects{/host}{/owner}{/repository}",
```

Comment:
since the domain name may change, I wonder whether this should all live under a config file and shared across all sub-projects in the webapp repo?

Reply:
So do we not include the domain name here? Or automatically pull it from somewhere else

Reply:
yes pull it from a file in the repo. @azeemshaikh38 any suggestion?
Maybe do that in a follow-up PR and create a tracking issue for it.

Reply:
Leave it as `/projects/{host}/{owner}/{repository}` for now. That is generic enough to work for any domain. Leave a `TODO` to make this domain-specific in future.

Reply:
my comment was about `api.securityscorecards.dev` though, not the GET request.
If we spin our clusters to answer on all domains, it's fine. But it we stick to a single, having the domain ina config file is better